README.md

```diff
@@ -62,7 +83,7 @@ author.save(function(err){
 What is module.exports and why do we use it?
 
 ```text
-Your answer...
+it is a JS convention that allows us to separate our JS files in a way that when we export it, we make their contents available to other files by requiring it in the appropriate file.
 ```
 
```

Comment:
It a command in node rather than a convention, but the rest of this is basically correct. We have to `module.exports` something specifically to make it available for `require()`/importing